src/_includes/components.njk

```diff
@@ -32,4 +32,27 @@
 <a href={{platform.url}} class="text-3xl hover:bg-white hover:bg-opacity-25">
 	<img src="images/icon_{{platform.name}}.png" alt class="w-social_media_icon m-3 transform hover:scale-105">
 </a>
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro sponsor_tier(sponsors, lg=0, md=0, sm=0) %}
```

Comment:
Maybe it would be better to have a single `size` parameter that takes in `lg`, `md`, or `small`? Or maybe we could refactor this into three different macros seeing as there isn't really any shared code between each case.